src/DssDeploy.t.sol

```diff
@@ -40,6 +40,14 @@ contract DssDeployTest is DssDeployTestBase {
         dssDeploy.deployPause(0, address(authority));
     }
 
+    function testFailMissingPause() public {
+        dssDeploy.deployVat();
+        dssDeploy.deployDai(99);
+        dssDeploy.deployTaxation();
+        dssDeploy.deployAuctions(address(gov));
+        dssDeploy.deployShutdown();
```

Comment:
Actually not sure why this is failing.  The `deployShutdown()` isn't checking for the `pause`.  It's failing because of a check on the `cat`.

Reply:
what the test intents to show is that if you follow the chain but you miss the pause step, you can't deploy the esm one.